modules/data.atmosphere/inst/integrationTests/test.download.AmerifluxLBL.R

```diff
@@ -0,0 +1,165 @@
+library(testthat)
+
+tmpdir <- tempfile(pattern = "amerifluxData")
+dir.create(tmpdir)
+on.exit(teardown(unlink(tmpdir, recursive = TRUE)))
```

Comment:
FYI `teardown()` is superseded.  It is now recommended to use the `withr` package for this kind of thing.  This is still valid code though and will continue to work.